recipes/r-rsqlite/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-rsqlite
+  version: "1.1_2"
+
+source:
+  fn: RSQLite_1.1-2.tar.gz
+  url:
+    - http://cran.r-project.org/src/contrib/RSQLite_1.1-2.tar.gz
+    - http://cran.r-project.org/src/contrib/Archive/RSQLite/RSQLite_1.1-2.tar.gz
+
+build:
+
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - r-base
+    - r-bh
+    - r-dbi >=0.4_9
+    - r-rcpp
+    - r-memoise
+    - r-plogr
+    - posix               # [win]
+    - {{native}}toolchain # [win]
```

Comment:
I think you need here two spaces before the `#`.
How do you generate these recipes? With a skeleton? If so please update your conda-build.
I have also a script to assist you if you like.